build/merge-coverage.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-env node */
+const fs = require('fs');
+
+// Jest writes coverage after the process exits
+// so we need to poll and wait for all coverage files.
+const waitForAllCoverage = config => {
+  return Promise.all(
+    config.environments.map(
+      env =>
+        new Promise(resolve => {
+          const interval = setInterval(() => {
+            const coverageExists = fs.existsSync(
```

Comment:
I thought exists was being deprecated?

Reply:
It looks like exists() is being deprecated, but existsSync() is not.